Changed title to Trainer Training 2020 (#612)
carpentries/carpentries.org: content/support/membership/membership-faq.md

  nav: membership
aliases:
- /member_faq/
- /partnership_faq/
---

> ## Memberships

Reviewer: ```suggestion
## Partnerships
```

Reviewer 2: Whoops! Thanks @elletjies!

- **Hands-on, practice-based training in teaching and lesson development.** Receive priority access to our Instructor Training and Collaborative Lesson Development Training programs, which will empower high-quality lesson creation and instruction at your institution.
- **Computational and data skills training.** Gain access to our global network of thousands of certified Instructors, who will deliver a hands-on learning experience targeting novice learners across a broad range of disciplines.
- **Professional development.** Keep your team's skills relevant to the evolving digital workplace with short-format training in remote collaboration, effective meeting facilitation, and more.
- **Values-driven community of practice.** Become part of our inclusive community, centered on building strength through diversity and empowering community collaboration through lifelong learning.
{{< /accordion >}}

{{< accordion title="What are the standard Partnership packages you offer?" >}}
We offer Launch, Expand, Scale, and Transform Partnership packages, each with an allotted number of service options that can be used across our programming. 

> No matter which level of Partnership you pursue, The Carpentries team with help you mix and match our service offerings to build the best training program for your team!

Reviewer 3: ```suggestion
No matter which level of Partnership you pursue, The Carpentries team will help you mix and match our service offerings to build the best training program for your team!
```

Read more about the benefits for each plan on our [Partnership](/partnership) page.
{{< /accordion >}}

{{< accordion title="Which Partnership should I purchase?" >}}
- **Launch Partnerships** are most appropriate for organisations in the early stages of establishing a local community of practice around data science and computational research skills. This Partnership level can be used to demonstrate the uniqueness and impact of The Carpentries teaching model, as a stepping-stone towards securing sustained organisational investment.

> - **Expand Partnerships** are for organisations ready to increase their training capacity, and provides multiple options for skill development and community building.

Reviewer 3: ```suggestion
- **Expand Partnerships** are for organisations ready to increase their training capacity, and provide multiple options for skill development and community building.
```

- **Scale Partnerships** are designed for organisations seeking to broaden their impact. This level enables expanded programming for long-lasting growth of overall institutional capacity.

> - **Transform Partnerships** are intended for organisations pursuing truly institution-wide change. This level enables accelerated large-scale capacity growth and sustained, organisation-wide impact. 

Reviewer 3: ```suggestion
- **Transform Partnerships** are intended for organisations pursuing even broader institution-wide change. This level enables accelerated large-scale capacity growth and sustained, organisation-wide impact. 
```

{{< /accordion >}}

{{< accordion title="What if those Partnership packages do not fit my organisations’s needs?" >}}
We would be happy to talk with you to learn more about your organisation’s goals and create a custom plan that works for you.  Contact our Partnership team to schedule a meeting at [{{< param membership_email >}}](mailto:{{< param membership_email >}}).
{{< /accordion >}}

{{< accordion title="Do I need to renew my Partnership each year?" >}}
Yes. The Carpentries Partnerships are annual subscriptions, extending for a period of one year. Renewing your Partnership in advance is a great way to avoid a lapse in services.
{{< /accordion >}}

{{< accordion title="Are any discounts available?" >}}
Absolutely! Our impact depends on continuing contributions from our community. To recognise those contributions, we offer discounts on Partnership packages to organisations with members actively contributing in a variety of roles. Information about available discounts can be found on our [pricing page](https://carpentries.org/support/pricing/#service-discounts).
{{< /accordion >}}

{{< accordion title="How can I find out if my institution already has a Partnership?" >}}

> A list of our [current Partner organisations](/partners/) can be found on our website. If you would like

Reviewer 3: ```suggestion
A list of our [current Partner Organisations](/partners/) can be found on our website. If you would like
```
